DataFormats/L1TGlobal/interface/AXOL1TLScore.h

```diff
@@ -0,0 +1,71 @@
+#ifndef DataFormats_L1TGlobal_AXOL1TLScore_h
+#define DataFormats_L1TGlobal_AXOL1TLScore_h
```

Comment:
Because the score is saved as an output of the GlobalProducer/L1TGlobal, I put it with the other GlobalProducer/L1TGlobal outputs in the DataFormats/L1TGlobal directory

Reply:
Okay, thanks.